kytos/utils/client.py

```diff
@@ -78,6 +78,34 @@ def get_napp(self, username, name):
                                  username, name, res.status_code, res.reason)
         return json.loads(res.content)
 
+    def reload_napps(self, napps=None):
+        """Reload a specific NApp or all Napps.
+
+        Args:
+            napp (list): NApp list to be reload.
+        Raises:
+            requests.HTTPError: When there's a server error.
+
+        """
+        if napps is not None:
+            for napp in napps:
+                api = self._config.get('kytos', 'api')
+                endpoint = os.path.join(api, 'api', 'kytos', 'core', 'reload',
+                                        napp[0], napp[1])
+                response = self.make_request(endpoint)
+        else:
+            api = self._config.get('kytos', 'api')
+            endpoint = os.path.join(api, 'api', 'kytos', 'core', 'reload',
+                                    'all')
+            response = self.make_request(endpoint)
```

Comment:
You can rewrite this with better readability by changing the order, and remove the else:
```
if napps is None:
...

for napp in napps:
....
```